libkbfs/interfaces.go

```diff
@@ -65,7 +65,8 @@ type diskBlockCacheGetter interface {
 }
 
 type diskBlockCacheSetter interface {
-	SetDiskBlockCache(DiskBlockCache)
+	ResetDiskBlockCache(ctx context.Context) error
```

Comment:
Slight preference for keeping the interface to a minimum, since we can easily re-add this function whenever we do need it.  But I don't care that much, so if you have plans for this in the near future, I'm fine with keeping it.

(Either way I'm happy to keep the logic split off into a reusable `*Locked` function.)